apps/web/src/scenes/Post/StandalonePostView.tsx

```diff
@@ -45,21 +46,23 @@ export default function StandalonePostView({ postRef, queryRef }: Props) {
   const router = useRouter();
   const commentId = router.query.commentId as string;
 
-  if (commentId) {
-    showModal({
-      content: (
-        <PostCommentsModal
-          fullscreen={isMobile}
-          postRef={post}
-          queryRef={query}
-          activeCommentId={commentId}
-        />
-      ),
-      isFullPage: isMobile,
-      isPaddingDisabled: true,
-      headerVariant: 'standard',
-    });
-  }
+  useEffect(() => {
```

Comment:
turns out we were simply calling `showModal` directly in the render logic instead of having it in a `useEffect`. every time the component re-rendered, we'd stack on an additional modal!